Notebooks/test new class.ipynb

```diff
@@ -0,0 +1,276 @@
+{
```

Comment:
I think we moved all the examples out into the cookbook, so this should probably go there.